integration/src/main/java/scripts/utils/WorkspaceAllocateTestScriptBase.java

```diff
@@ -28,6 +30,13 @@ public abstract class WorkspaceAllocateTestScriptBase extends WorkspaceApiTestSc
   private UUID workspaceUuid;
   private String spendProfileId;
 
+  public static void makeProperties(Properties properties) {
```

Comment:
sorry we have some miscommunication here.

i was thinking you have a static field DEFAULT_PROPERTIES here and you will assert that in `Cloneworkspace`. Your assertion is better though. Can you remove this makeProperties method and just set the properties in createWorkspace?